factory/amf_config_test.go

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Canonical Ltd.
```

Comment:
To use REUSE default format (https://reuse.readthedocs.io/en/stable/usage.html#annotate)
```suggestion
// SPDX-FileCopyrightText: 2024 Canonical Ltd.
```